### ISSUE TITLE
[Question] Does this still work?

### ISSUE THREAD

**Issue by Hexality** [itsfolf/Discord-Screenshare-Linux#16]
I'm searching for a way to enable screen sharing without a custom client, so I came to ask if this still works or if you have any other recommendation regarding this.

**Comment by itsfolf** [comment]
No, discord updated their binary which broke this, the binaries are now too different to practically memory patch. A patch is still possible but you'd need to reimplement all of the video logic including rtc and udp mediasockets, and patch discord_voice js to use the custom addon.
